### PR TITLE
Convert isipv6 to bool since it is used as one

### PR DESCRIPTION
**Pull request by AtariDreams**
I also removed an unused hw_tls struct member that I found was completely unused.

### CODE REVIEW

File: sys/netinet/tcp_stacks/tcp_rack.h [hunk 1]
```diff
@@ -340,9 +340,6 @@ struct rack_fast_send_blk {
 	struct udphdr *udp;
 	struct mbuf *m;
 	uint32_t o_m_len;
-	uint32_t rfo_apply_push : 1,
```
Comment (bsdimp) [inline sys/netinet/tcp_stacks/tcp_rack.h:343]: These are still used in the tree.

Comment (gAlfonso-bit) [reply sys/netinet/tcp_stacks/tcp_rack.h:343]: Alright

**Comment by gAlfonso-bit** [comment]
Fixed

**Comment by emaste** [comment]
headline says `hw_tls` but the change is to `isipv6`

**Pull request closed without merging**